### PR TITLE
syncer: add skip-dmls document

### PR DESCRIPTION
**Pull request by holys**
@GregoryIan  @WangXiangUSTC  @lilin90  PTAL

### CODE REVIEW

File: tools/syncer.md [hunk 1]
```diff
@@ -73,25 +73,44 @@ batch = 10
...
+# 
+# skip all delete statements in all foo.* tables.
+# [[skip-dmls]]
+# db-name = "foo"
+# type = "delete"
 
 # Support whitelist filter. You can specify the database and table to be synchronised. For example:
 # Synchronise all the tables of db1 and db2:
-replicate-do-db = ["db1","db2"]
+# replicate-do-db = ["db1","db2"]
```
Comment (holys) [inline tools/syncer.md]: Comment it because it is optional configurations. Same as follows

**Comment by queenypingcap** [comment]
@lilin90 PTAL.

File: tools/syncer.md [hunk 2]
```diff
@@ -73,25 +73,44 @@ batch = 10
 # The testing address for pprof. It can also be used by Prometheus to pull the syncer metrics.
 status-addr = ":10081"
 
-skip-sqls = ["ALTER USER", "CREATE USER"]
+# Notice: skip-sqls is abandoned, please use skip-ddls instead.
```
Comment (lilin90) [inline tools/syncer.md]: Update this sentence to:

"Notice: skip-sqls is abandoned, and use skip-ddls instead."

Note: Do not use the word "please" in user documents, which is over-polite.

File: tools/syncer.md [hunk 3]
```diff
@@ -73,25 +73,44 @@ batch = 10
...
+# skip-ddls skip ddl statements which supports regular expression. When there are DDL sqls incompatible with tidb, you should skip them via these rules.
```
Comment (lilin90) [inline tools/syncer.md]: Change this sentence to:
"skip-ddls skips DDL statements which support regular expressions. If the DDL sqls are incompatible with TiDB, skip them using these rules."

File: tools/syncer.md [hunk 4]
```diff
@@ -73,25 +73,44 @@ batch = 10
...
+# skip-ddls = ["^CREATE\\s+USER"]
+
+# Notice: skip-events is abandoned, please use skip-dmls instead
```
Comment (lilin90) [inline tools/syncer.md]: Change to:
"Notice: skip-events is abandoned, and use skip-dmls instead."

File: tools/syncer.md [hunk 5]
```diff
@@ -73,25 +73,44 @@ batch = 10
...
+# skip-dmls skip DML statements. type value can be 'insert', 'update', 'delete'.
```
Comment (lilin90) [inline tools/syncer.md]: -> skip-dmls skips DML statements. The type value can be 'insert', 'update' and 'delete'.

**Comment by holys** [comment]
@lilin90 PTAL again

**Comment by lilin90** [comment]
LGTM

**Pull request merged**